isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0"?>
 <application name="jigsaw" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
   <brief>
-    Improves camera pointing and a whole lot more!
+    Improves image alignment
```

Comment:
I think a little more might be nice in this section. Maybe "through updates to kernel information"?